applications/ray/kuberay-tpu-webhook/deployments/mutate-pod-webhook-cfg.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: admissionregistration.k8s.io/v1
 kind: MutatingWebhookConfiguration
 metadata:
-  name: kuberay-tpu-webhook-cfg
+  name: mutate-pod-webhook-cfg
```

Comment:
This should stay as `kuberay-tpu-webhook-cfg` or similar. A webhook named `mutate-pod-webhook-cfg` will be misleading since it doesn't have the word `kuberay` or `tpu` in it

Reply:
Done.